### ISSUE TITLE
Mark reviewed link doesn't always work when reviewing stale knowls

### ISSUE THREAD

**Issue by AndrewVSutherland** [LMFDB/lmfdb#4158]
Clicking the "Mark reviewed" link on https://beta.lmfdb.org/knowledge/review_stale often does nothing (running in chrome).  Using "open link in new tab" seems to work.  I have no idea what could be causing this, but if there is an easy fix it would be nice to fix.

**Comment by edgarcosta** [comment]
@AndrewVSutherland clicking on the review button it triggers an async request to mark the knowl as reviewed. 
The only change you should on https://beta.lmfdb.org/knowledge/review_stale is "Positively reviewed! (Back to beta)"

**Comment by AndrewVSutherland** [comment]
@edgarcosta Yes, but often that change never happens (feel free to try it yourself, you can always revert it back to beta when you are done).

**Comment by edgarcosta** [comment]
You are right. I'm honestly confused, as when I tested it, I even checked that async request went through... but now the knowl I reviewed is back in beta... I'm indeed confused.
I will try to debug this tonight or tomorrow over LMFDB fridays.

**Comment by edgarcosta** [comment]
I think #4160 just fixed this.
However, there is some javascript bug. If the inline diff is open, the knowl gets reviewed, but the button doesn't change.

**Comment by roed314** [comment]
Has this been resolved?

**Comment by AndrewVSutherland** [comment]
The issue Edgar noted still occurs (I tested it just now).  To reproduce, go to a stale know, click "inline", then click "Mark reviewed" (the knowl is marked as reviewed but there is no indication of this on the screen).

I note that it works correctly if you click "side by side" rather than "inline".

**Comment by roed314** [comment]
Interesting.  Works for me on Firefox.

**Comment by AndrewVSutherland** [comment]
I see the issue in both chrome and firefox (running on Ubuntu).

**Comment by roed314** [comment]
Ok, it worked earlier but not just now.  Anyway, I'm going to focus on other issues.
